docs/tempo/website/operations/important_metrics.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Important metrics
+---
+
+## Important Metrics
+
+### Free Disk Space
```

Comment:
I didn't expect to see this on important metrics! Here we're giving the user instructions on how to increase the disk space. 

Maybe we can list && explain the metrics here, and put the guide/procedure in some other place.

Reply:
Yeah agreed, this is just a copy n paste from a different document, will move this to `ingester_pvcs.md`